triedb/hashdb/database.go

```diff
@@ -114,15 +114,8 @@ type Config struct {
 	ReferenceRootAtomicallyOnUpdate bool   // Whether to reference the root node on update
 }
 
-func (c Config) BackendConstructor(diskdb ethdb.Database, config *triedb.Config) triedb.DBOverride {
-	var resolver ChildResolver
-	if config.IsVerkle {
-		// TODO define verkle resolver
-		log.Crit("Verkle node resolver is not defined")
```

Comment:
why did we remove this? we'll want to pass the correct option when verkle has a trie resolver type we can use here.

Reply:
Removed following your comment [here](https://github.com/ava-labs/coreth/pull/820#discussion_r2010989796) 😉 
IMO we can always re-add this later, once it's supported.